### ISSUE TITLE
docker container not starting

### ISSUE THREAD

**Issue by ragnar76** [martin-ueding/geo-activity-playground#123]
Just tried to update and docker container failes to start. I did the usual thing:

- cd geo-activity-playground
- git pull
- rm -rf /var/www/tracks/gpx-tracks/Cache/*
- docker build -t geo-activity-playground .
- docker run -p 5000:5000 -v /var/www/tracks/gpx-tracks:/data -it geo-activity-playground

Which gives me this error:

```
root@dev:~/geo-activity-playground # docker run -p 5000:5000 -v /var/www/tracks/gpx-tracks:/data -it geo-activity-playground
2024-07-11 20:35:51 geo_activity_playground.core.cache_migrations INFO Apply cache migration if needed …
2024-07-11 20:35:51 geo_activity_playground.core.cache_migrations INFO Applying cache migration add_consider_for_achievements …
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "/app/geo_activity_playground/__main__.py", line 118, in <module>
    main()
  File "/app/geo_activity_playground/__main__.py", line 93, in main
    options.func(options)
  File "/app/geo_activity_playground/__main__.py", line 67, in <lambda>
    *make_activity_repository(options.basedir, options.skip_strava),
     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/geo_activity_playground/__main__.py", line 100, in make_activity_repository
    apply_cache_migrations()
  File "/app/geo_activity_playground/core/cache_migrations.py", line 119, in apply_cache_migrations
    migration()
  File "/app/geo_activity_playground/core/cache_migrations.py", line 82, in add_consider_for_achievements
    df = pd.read_parquet("Cache/activities.parquet")
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pandas/io/parquet.py", line 667, in read_parquet
    return impl.read(
           ^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pandas/io/parquet.py", line 267, in read
    path_or_handle, handles, filesystem = _get_path_or_handle(
                                          ^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pandas/io/parquet.py", line 140, in _get_path_or_handle
    handles = get_handle(
              ^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/pandas/io/common.py", line 882, in get_handle
    handle = open(handle, ioargs.mode)
             ^^^^^^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: 'Cache/activities.parquet'
```

**Comment by ragnar76** [comment]
I was able to get it back working by using an older version, create the cache files and switch back to the latest git commit.

**Comment by martin-ueding** [comment]
The problem is that the migration didn't anticipate that the file didn't exist yet. I've fixed that in 0.24.0
